.travis.yml

```diff
@@ -1,4 +1,7 @@
 language: ruby
+before_install:
+  - "yes | gem uninstall -i /home/travis/.rvm/gems/ruby-2.5.3@global bundler"
```

Comment:
Need to uninstall bundler v2+ and install V2- due to known travis bundler bug https://github.com/travis-ci/travis-ci/issues/1176

Reply:
To me, `ruby-2.5.3` seems to be hardcoded. Maybe it should be dynamic, without any fixed ruby version.

Why not use approach suggested in [Travis's documentation? ](https://docs.travis-ci.com/user/languages/ruby/#bundler-20) Haven't tested though, but I think that simple  `gem install bundler` in `before_install` would to the trick.

Reply:
Yep, that's hardcoded and ad-hoc AF, but simple `gem install bundler` installs latest bundler and it brokes the entire build =(.
That's the command Travis console suggested for reinstall.

Reply:
I'm having hard time figuring out what exactly defines that we have to use bundler 1.x since no `Gemfile.lock` is present, rubygems itself? But w/e, let it be.

Can we modify existing command not to be dependent on ruby version? Using `$(rvm gemdir)@global` should do the trick.

Reply:
I'll check this out.